tools/debugging/stress_test_transfers.py

```diff
@@ -523,6 +524,32 @@ def run_profiler(
     return profiler_processes
 
 
+def get_balance_for_node(node: RunningNode) -> TokenAmount:
+    response = requests.get(f"{node.url}/api/v1/channels").json()
```

Comment:
I'd suggest using the response without the json first. If an internal server error happened, then the response will be plain text, and not a json, this would then raise an exception.